ui/static/js/nav_handler.js

```diff
@@ -129,6 +129,20 @@ class NavHandler {
         }
     }
 
+    unsubscribeFromFeed() {
+        let unsubscribeLinks = document.querySelectorAll("[data-action=remove-feed]");
+        if (unsubscribeLinks.length === 1) {
```

Comment:
If there were to be multiple `remove-feed` links on the same page in the future, this checks stop the feature right away, making it deterministic in which one of the feeds to remove (none).

Reply:
Then, what about using `querySelector()` to fetch only one element instead of many?

Reply:
I thought that there may be multiple links in the future, with different targets, like below:

```
<a href="#" data-action="remove-feed">Remove feed A</a>
<a href="#" data-action="remove-feed">Remove feed B</a>
```

then, I'd like this defense mechanism to kick in, disallowing any of the feeds to be removed (because it's not clear which of the links the user wants).

If we were to use `querySelector()`, we would always get the first link, which may or may not be the one the user was looking at, which sounds very dangerous.